xskillscore/core/contingency.py

```diff
@@ -0,0 +1,787 @@
+            {
+                OBSERVATIONS_NAME + '_category': yes_category,
+                FORECASTS_NAME + '_category': no_category,
+            },
+            drop=True,
+        )
+        false_alarms = self.table.sel(
+            {
+                OBSERVATIONS_NAME + '_category': no_category,
+                FORECASTS_NAME + '_category': yes_category,
+            },
+            drop=True,
+        )
+
+        return (hits * correct_negs) / (misses * false_alarms)
+
+    @dichotomous_only
+    def odds_ratio_skill_score(self, yes_category=2):
+        """
+            Returns the odds ratio skill score(s) for dichotomous contingency data
+
+            Parameters
+            ----------
+            yes_category : value, optional
+                The category coordinate value of the category corresponding to 'yes'
+
+            Returns
+            -------
+            xarray.Dataset or xarray.DataArray
+                An array containing the equitable odds ratio skill score(s)
+
+            Notes
+            -----
+            See http://www.cawcr.gov.au/projects/verification/
+        """
+
+        no_category = abs(yes_category - 2) + 1
+
+        hits = self.table.sel(
+            {
+                OBSERVATIONS_NAME + '_category': yes_category,
+                FORECASTS_NAME + '_category': yes_category,
+            },
+            drop=True,
+        )
+        correct_negs = self.table.sel(
+            {
+                OBSERVATIONS_NAME + '_category': no_category,
+                FORECASTS_NAME + '_category': no_category,
+            },
+            drop=True,
+        )
+        misses = self.table.sel(
+            {
+                OBSERVATIONS_NAME + '_category': yes_category,
+                FORECASTS_NAME + '_category': no_category,
+            },
+            drop=True,
+        )
+        false_alarms = self.table.sel(
+            {
+                OBSERVATIONS_NAME + '_category': no_category,
+                FORECASTS_NAME + '_category': yes_category,
+            },
+            drop=True,
+        )
+
+        return (hits * correct_negs - misses * false_alarms) / (
+            hits * correct_negs + misses * false_alarms
+        )
+
+    def accuracy(self):
+        """
+            Returns the accuracy score(s) for a contingency table
+
+            Returns
+            -------
+            xarray.Dataset or xarray.DataArray
+                An array containing the accuracy score(s)
+
+            Notes
+            -----
+            See http://www.cawcr.gov.au/projects/verification/
+        """
+
+        hits = self.table.where(
+            self.table[OBSERVATIONS_NAME + '_category']
+            == self.table[FORECASTS_NAME + '_category']
+        ).sum(
+            dim=(OBSERVATIONS_NAME + '_category', FORECASTS_NAME + '_category'),
+            skipna=True,
+        )
+        N = self._sum_categories('total')
+
+        return hits / N
+
+    def Heidke_score(self):
```

Comment:
Even though a person's name I would follow PEP convention and do `heidke_score`. You can see that the Github syntax coloring even thinks it's a class due to the capitalization.